sites/layouts/documentation_pattern.html

```diff
@@ -0,0 +1,700 @@
+													]
+												}
+											]
+										}
+									]
+								},
+								{
+									"template": "[data-code-compare]",
+									"test": "fn:guessType",
+									"value": "/sample",
+									"expect": "code-compare",
+									"mapping": [
+										{ "selector": "figcaption[data-rev]", "value": "/rev_title" },
+										{ "selector": "code[data-rev]", "value": "/rev_code", "isHTML": true },
+										{ "selector": "figcaption[data-from]", "value": "/from_title" },
+										{ "selector": "code[data-from]", "value": "/from_code", "isHTML": true }
+									]
+								}
+
+							]
+						}
+					]
+				}
+			]
+		},
+
+		{ "selector": "[data-history]", "value": "/history",
+			"mapping": [
+				{ "selector": "li", "value": "/{{lng}}" }
+			]
+		}
+
+	],
+	"trigger": true
+}'>
+
+	<template>
+		<h3>
+			<span data-name>[[ Name of the variation ]]</span>
+			<template data-status-stable>
+				<span class="label label-success mrgn-lft-sm"><span class="wb-inv">Status: </span>Stable</span>
+			</template>
+			<template data-status-tobereviewed>
+				<span class="label label-info mrgn-lft-sm"><span class="wb-inv">Status: </span>To be reviewed</span>
+			</template>
+			<template data-status-provisional>
+				<span class="label label-warning mrgn-lft-sm"><span class="wb-inv">Status: </span>Provisional</span>
+			</template>
+			<template data-status-deprecated>
+				<span class="label label-default mrgn-lft-sm"><span class="wb-inv">Status: </span>Deprecated</span>
+			</template>
+		</h3>
+
+		<p data-description>[[ Short description ]]</p>
+		{%- if lng == "fr" -%}
+		<p>Consultez les <a data-guidance href>guides de conception concernant cette variante</a>.</p>
+		{%- else -%}
+		<p>Consult the <a data-guidance href>design guidance about this variation</a>.</p>
+		{%- endif -%}
+
+		<!--
+		<p>TODO: Screen capture</p>
+		<p>TODO: Working example link</p>
+		-->
+		<template data-variation-example>
+			<h4>{% if lng == "fr" %}Exemple(s) pratique(s){% else %}Working example(s){% endif %}</h4>
+			<ul>
+				<template>
+					<li><a href></a></li>
+				</template>
+			</ul>
+		</template>
+
+		<template data-implementation>
+			<h4>{% if lng == "fr" %}Implémentation{% else %}Implementation{% endif %}</h4>
+
+			<div class="wb-tabs" data-implementation-tabs>
+				<template>
+					<div class="tabpanels">
+						<details>
+							<summary>Name of the implementation</summary>
+							<p data-introduction>Introduction</p>
+
+							<template data-example>
+								<p>Working examples:</p>
+								<ul>
+									<template>
+										<li><a href></a></li>
+									</template>
+								</ul>
+							</template>
+
+							<template data-instructions>
+								<p>Instructions:</p>
+								<ul>
+									<template>
+										<li>Instruction</li>
+									</template>
+								</ul>
+							</template>
+
+							<template data-tech-considerations>
+								<p>Technical considerations:</p>
```

Comment:
Please translate in French
```suggestion
								<p>{% if lng == "fr" %}Considération techniques {% else %}Technical considerations{% endif %}:</p>
```